docs/en-US/component/radio.md

```diff
@@ -59,14 +67,15 @@ radio/with-borders
 
 ### Radio Attributes
 
-| Name                  | Description                          | Type                                     | Default |
-| --------------------- | ------------------------------------ | ---------------------------------------- | ------- |
-| model-value / v-model | binding value                        | ^[string] / ^[number] / ^[boolean]       | —       |
-| label                 | the value of Radio                   | ^[string] / ^[number] / ^[boolean]       | —       |
-| disabled              | whether Radio is disabled            | ^[boolean]                               | false   |
-| border                | whether to add a border around Radio | ^[boolean]                               | false   |
-| size                  | size of the Radio                    | ^[enum]`'large' \| 'default' \| 'small'` | —       |
-| name                  | native `name` attribute              | ^[string]                                | —       |
+| Name                  | Description                                                            | Type                                     | Default |
+|-----------------------|------------------------------------------------------------------------|------------------------------------------|---------|
+| model-value / v-model | binding value                                                          | ^[string] / ^[number] / ^[boolean]       | —       |
+| value ^(2.5.2)        | the value of Radio                                                     | ^[string] / ^[number] / ^[boolean]       | —       |
```

Comment:
**Suggestion:** as we are adding new APIs, we might want to make it a minor patch as `^2.6.0`, adding new API is considered breaking.